reconcile/utils/terrascript_aws_client.py

```diff
@@ -850,21 +850,16 @@ def populate_iam_users(
             for user_policy in user_policies:
                 policy_name = user_policy["name"]
                 account_name = user_policy["account"]["name"]
-                account_uid = user_policy["account"]["uid"]
                 for user in users:
-                    # replace known keys with values
                     user_name = self._get_aws_username(user)
-                    policy = user_policy["policy"]
-                    policy = policy.replace("${aws:username}", user_name)
-                    policy = policy.replace("${aws:accountid}", account_uid)
 
                     # Ref: terraform aws_iam_policy
                     tf_iam_user = self.get_tf_iam_user(user_name)
                     identifier = f"{user_name}-{policy_name}"
                     tf_aws_iam_policy = aws_iam_policy(
                         identifier,
                         name=identifier,
-                        policy=policy,
+                        policy=user_policy["policy"],
```

Comment:
Doesn't this belong to https://github.com/app-sre/qontract-reconcile/pull/4177?

Reply:
Rebase error! Fixed.